drivers/clock_control/clock_control_silabs_siwx91x.c

```diff
@@ -35,8 +45,8 @@ static int siwx91x_clock_on(const struct device *dev, clock_control_subsys_t sys
 	switch (clockid) {
 	case SIWX91X_CLK_ULP_UART:
 		RSI_PS_UlpssPeriPowerUp(ULPSS_PWRGATE_ULP_UART);
-		RSI_ULPSS_UlpUartClkConfig(ULPCLK, ENABLE_STATIC_CLK,
-					   false, ULP_UART_ULP_MHZ_RC_CLK, 1);
+		RSI_ULPSS_UlpUartClkConfig(ULPCLK, ENABLE_STATIC_CLK, false,
+					   ULP_UART_ULP_MHZ_RC_CLK, 1);
```

Comment:
I believe the original code was fine.

If you still believe it need to be changes, place place this change in a separated commit (so the reviewer can focus to the important changes).